webapp/channels/src/components/interactive_dialog/interactive_dialog.tsx

```diff
@@ -98,13 +110,11 @@ export default class InteractiveDialog extends React.PureComponent {
             callback_id: callbackId,
             state,
             submission: values,
-        };
+        } as unknown as DialogSubmission;
```

Comment:
Why this needs this type conversion here? shouldn't be providing the data needed for the DialogSubmission instead of forcing the type here?